README.md

```diff
@@ -171,6 +174,6 @@ chmod u+x /home/mes-aides-$BRANCH/deploy.sh
 
 ```sh
 ssh-add ~/.ssh/mes-aides-bot
-ssh mes-aides-$BRANCH@sgmap.fr "PORT=8200 OPENFISCA_PORT=12200 ./deploy.sh $BRANCH"
+ssh mes-aides-$BRANCH@sgmap.fr "PORT=8200 OPENFISCA_PORT=12200 ./deploy.sh"
 ssh root@sgmap.fr "service nginx reload"
```

Comment:
`nginx reload` => look at the configuration files and try to apply changes in _zero downtime_ way.
`nginx restart` => kill the process and restart nginx

You don't want to use `nginx restart` in your deployment ;)
If you just update or create a configuration file, there will be no problem.